src/test/java/org/opensearch/security/auditlog/config/AuditConfigFilterTest.java

```diff
@@ -13,24 +13,18 @@
  *  permissions and limitations under the License.
  */
 
-package com.amazon.opendistroforelasticsearch.security.auditlog.config;
+package org.opensearch.security.auditlog.config;
 
-import com.amazon.opendistroforelasticsearch.security.auditlog.impl.AuditCategory;
-import com.amazon.opendistroforelasticsearch.security.support.ConfigConstants;
-import com.amazon.opendistroforelasticsearch.security.support.WildcardMatcher;
+import org.opensearch.security.auditlog.impl.AuditCategory;
+import org.opensearch.security.support.ConfigConstants;
+import org.opensearch.security.support.WildcardMatcher;
+import org.junit.Assert;
```

Comment:
There are lots of test class that add `import org.junit.Assert;` which is unnecessary.